IanGRap/Squish
# Bugfix: #65 (fonts are not unified)

Contributors: Austin

Goal:
- Change all of the fonts so that they are unified across all scenes

What changed:
- Changed all fonts to Butcherman (for titles) and Exo 2 (for subtitles or regular text)
- IMPORTANT: I made the UI in the Player Select screen a prefab so that it is easy to carry over between scenes.

How to test:
- Load up the player select screen and make sure the fonts are Butcherman and Exo 2
- Go to any working match scene (I used pause menu) and make sure the game over text is Butcherman and Exo 2
